pwiz_tools/Skyline/Controls/Graphs/SummaryRelativeAbundanceGraphPane.cs

```diff
@@ -388,6 +433,16 @@ private static bool IsStandard(PeptideDocNode pepDocNode)
         {
             return pepDocNode.GlobalStandardType != null;
         }
+        public void OnSuspendLayout(object sender, EventArgs eventArgs)
```

Comment:
The handler should be private. It can wrap a public no-arguments function that can be called from tests. Normally, I would use something like:

SuspendLayout(!Settings.Default.GroupComparisonSuspendLabelLayout)

With:

public void SuspendLayout(bool suspend)
{
    Settings.Default.GroupComparisonSuspendLabelLayout = suspend;
    if (!suspend)
...